justfile

```diff
@@ -20,14 +20,15 @@ lint *FLAGS: gomod
   golangci-lint run {{FLAGS}}
 
 generate:
-    go run github.com/Khan/genqlient api/genqlient.yaml
-    go generate ./...
+  go run github.com/Khan/genqlient api/genqlient.yaml
+  go generate ./...
 
 gomod:
-  #!/usr/bin/env bash
-  set -euo pipefail
+  #!/usr/bin/env sh
+  set -euf
+
   go mod tidy
-  git diff --no-ext-diff --exit-code go.mod go.sum
+  git diff -G. --no-ext-diff --exit-code go.mod go.sum
```

Comment:
```
       -G<regex>
           Look for differences whose patch text contains added/removed lines that match <regex>.
```

I assume `-G.` means “only show diffs that match a regexp of at least one character”, making it ignore metadata-only changes like file permissions?

Reply:
Yep, I should have documented it in a comment, but I want to remove this ASAP